### ISSUE TITLE
luapp and lua-cairo support

### ISSUE THREAD

**Issue by npyl** [Conky-for-macOS/conky-for-macOS#1]
I haven't yet been able to enable enable luapp and lua-cairo

**EDIT:** other than that, basic lua support such as `lua_load()` etc exists.

**Comment by npyl** [comment]
The only problem now seems to be getting `luapp` and `lua-cairo` to work on macOS.

**Comment by npyl** [comment]
`luapp` has been incorporated into conky in bb8723dd361d15be0077ab9270292b96572415e6.
This has enabled `luapp` for macOS, too!
`Cairo` also works (tested) but it seems that it is quite buggy.
Once I know more I will update this! 🤘

**Comment by npyl** [comment]
Progress:
I downloaded upstream `cairo`, built and installed it (Homebrew does not provide a cairo with xlib support, thus we need to compile it ourselves).
`cmake` was able to find it and it compiled successfully.
Then I had to copy `libcairo.dylib` (generated by conky) in a directory called `lua` in the same directory that conky is located.

This means that we can now start shipping MC with a conky with cairo support, but there are still some small problems such as `current directory` passed to conky (when called by MC) not containing `lua/libcairo.dylib`
